03_letters/shahriys/main_test.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"reflect"
+	"testing"
+)
+
+func TestSortLettersMain(t *testing.T) {
+	var bufletter bytes.Buffer
+	outletter = &bufletter
+
+	main()
+	expected := `a:2
+b:1`
+	actual := bufletter.String()
+
+	if expected != actual {
+		t.Errorf("Unexpected output in main()" + expected + " " + actual)
+	}
+
+}
+func TestSortLetters(t *testing.T) {
```

Comment:
I think it is best to test each of these two functions in isolation, rather than testing the output of concatenating the two function calls together. Can you please break this down to:

```go
func TestLetters(t *testing.T) {
  ....
}

func TestSortLetters(t *testing.T) {
  ....
}
```

Reply:
added